modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1106,7 +1107,10 @@ private CompletableFuture<?> createTableLocally(long causalityToken, String name
                 partitions, clusterNodeResolver, txManager, tableStorage, txStateStorage, replicaSvc, clock);
 
         // TODO: IGNITE-16288 directIndexIds should use async configuration API
```

Comment:
Looks like this TODO is not relevant anymore

Reply:
Fix it